auditbeat/docs/getting-started.asciidoc

```diff
@@ -212,7 +212,7 @@ include::{libbeat-dir}/docs/shared-template-load.asciidoc[]
 
 include::{libbeat-dir}/docs/dashboards.asciidoc[]
 
-[id="{beatname_lc}-starting"]
+[id="starting"]
```

Comment:
Just curious: Is this the same thing as `[[starting]]`?

Reply:
Yes, I was just doing a quick search/replace. Both syntaxes are valid.

Reply:
Cool!